### PR TITLE
make docs more clear about how to work with the current api

### PR DESCRIPTION
**Pull request by SantoDE**
cc @containous/traefik :)

### CODE REVIEW

**Comment by timoreimann** [comment]
Should we update the sample file too?

**Comment by emilevauge** [comment]
Ping @SantoDE ;)

**Comment by ldez** [comment]
@SantoDE Do you have time to update the sample file?

**Comment by SantoDE** [comment]
Hey @timoreimann & @ldez ,

done ;-)

**Comment by ldez** [comment]
@SantoDE  can you change the PR from v1.2 to master

**Comment by SantoDE** [comment]
@ldez done :)

**Comment by timoreimann** [comment]
Update of `traefik.sample.toml` still outstanding?

**Comment by SantoDE** [comment]
Ah, yeah! :D Sorry. we have to many examples 👼 

done @timoreimann

File: traefik.sample.toml [hunk 1]
```diff
@@ -996,43 +996,43 @@
 #
 # Optional
 #
-#[rancher]
```
Comment (timoreimann) [inline traefik.sample.toml]: I think we want to leave all ranger configuration parameters commented out, as with the other providers in the sample file.

AFAIU, the idea is that users copy the file and uncomment just what they need.

See also #1455.

**Pull request merged**